src/index.mjs

```diff
@@ -65,16 +65,28 @@ export function listen(options) {
 
   const timeoutFn = options.timeoutFn || requestIdleCallback;
 
+  // ray test touch <
```

Comment:
Could we remove the `ray test touch` lines from source now that we're closer to a solution we think is workable?

Reply:
Yes for sure, I think it's time.

Reply:
Done